projects/questionset-editor-library/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@project-sunbird/sunbird-questionset-editor",
-  "version": "5.5.0-beta.6",
+  "version": "5.6.1-beta.0",
   "dependencies": {
     "tslib": "^2.0.0"
   },
```

Comment:
@rajnishdargan Why these dependencies `"@angular/common": "~12.2.16", "@angular/core": "~12.2.16", ` still pointed to 12 version. These dependencies should be upgraded to 13 right ?

Please check and update the same .

Reply:
@vaibhavbhuva updated the dependencies to angular 13